website/docs/add-secure-apps/flows-stages/flow/inspector.md

```diff
@@ -14,13 +14,13 @@ As shown in the screenshot below, the flow inspector displays next to the select
 Be aware that when running a flow with the inspector enabled, the flow is still executed normally. This means that for example, a [User write](../stages/user_write.md) stage _will_ write user data.
 :::
 
-### Permissions and debug mode
-
-By default, the inspector is only enabled when the currently authenticated user is a superuser, OR if a user has been granted the [permission](../../../users-sources/access-control/permissions.md) **Can inspect a Flow's execution** (or is a user assigned to role with the permission).
+The inspector is accessible to users that have been granted the [permission](../../../users-sources/access-control/permissions.md) **Can inspect a Flow's execution**, either directly or through a role. Superuser can always inspect flow executions.
```

Comment:
```suggestion
The inspector is accessible to users that have been granted the [permission](../../../users-sources/access-control/permissions.md) **Can inspect a Flow's execution**, either directly or through a role. Superusers can always inspect flow executions.
```

Reply:
And we still want to add more, right, about how they also need View permissions on that specific flow?

Reply:
I think you're mixing that up with app entitlements, in this case they dont need any other permission